### ISSUE TITLE
Text input errors

### ISSUE THREAD

**Issue by EnricoBarbieri1997** [bitsongofficial/wallet-mobile#123]
Inputs are missing the design implementation for error tips and checking

**Comment by EnricoBarbieri1997** [comment]
Error text should be moved a couple of pixel up
![image](https://user-images.githubusercontent.com/12893767/188776961-b57a9ff2-5a1d-407d-a9d0-a590fe9ca1e6.png)

We can improve on it by making errorMessage a string or array. Only the first element of the array will be shown. In this way we can provide a list of errors for fields having more than one.

Furthermore Input implementation is fine but in some places a Search element is used for text fields and it doesn't use Input internally so you can't pass errorMessage props to those

**Comment by zheleznov163** [comment]
I'm aware of a problem with <Search />. Adding a new element was due to the need to use border when focusing on input. React Native styles do not implement the "box-sizing: content-box" CSS style, which causes the appearance of the box to be accompanied by a resizing of the element both externally and internally. This looks a little sloppy. In <Search />, I avoided this by adding a border with the background. I need some time to think of a generic component that will use different borders (focus, error) to mimic "content-box". If it's not critical now, it's easier to add the errorMessage functionality to <Search />, and later refactor and use one component
